### ISSUE TITLE
Improve screen reader support

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/WSUWP-Plugin-iDonate#136]
While testing out the accessibility of the plugin, I installed [ChromeVox](https://chrome.google.com/webstore/detail/chromevox/kgejglhpjiefppelpmljglcjbhoiplfn/related?hl=en) and started navigating through the form with a keyboard.

This article is also likely useful: https://inclusive-components.design/menus-menu-buttons/

I'm still learning, but a couple notes:

* When tabbing to a category and hitting enter, it seems like the focus should switch to the form rather than remaining on the button.
* When I navigate through and select a fund, then select an amount, the focus disappears. It should probably apply to the list of amounts under the fund selection tool.
* I'm not sure if the "Edit" links should have more information attached to them. Right now it's seen as an internal link. The "X" explains what's being removed, which is good.
* Focus skips over "Your generous donation will total $2050.00 today. When you proceed to...."
* Focus skips over "Is WSU in your will" and "Learn More" and goes straight to the will options.
* After proceeding to checkout, there is no focus on the next screen once it is successful. Hitting "Tab" does start Focus in the right place.
* Hitting enter on the question mark brings up a modal with no focus and does not provide information. The modal cannot be focused and cannot be closed with keyboard navigation.
* When first focusing on the payment option, the option being selected is not described.
* The Add Honor/Memory button triggers a modal but does not focus on the modal.
* Hitting Give now brings up a new screen, but does not focus or explain what happened.

I'm going to pause there. There are probably a few other things I missed that should be accounted for, but that will get things started.

**Comment by blairlierman** [comment]
@jeremyfelt Thanks for the report and the introduction to ChromeVox. We will continue to make more updates for accessibility.

**Comment by blairlierman** [comment]
Other accessibility issues that were discovered in testing:

**Plugin**
- Tabbing past the Proceed to Checkout button will put the focus on the PCI Compliance Requirements link, but not focus on the whole CC disclaimer paragraph.
- The Thank You page does not read the Thank You text, donation summary, billing information, or total amount charged.

**Embed**
- On the Payment Info screen, tabbing past the question mark will put the focus on the Payment Type drop down arrow, but I think it should go directly to the Payment Type button itself, like with the other How Often Will You Donate and Date buttons.
- Focusing on the Payment Type button does not read "Payment Type" aloud, so you're unable to determine what you're currently focused on. It also does not read the Payment Type options when you focus on them or select them.
- Tabbing past the Payment Type button will sometimes focus on the How Often Will You Donate button, and sometimes go directly into the How Often Will You Donate search field. When the latter happens it does not say which button you're focused on. It also does not read the How Often Will You Donate options when you focus on them or select them.
- Focusing on a date in the Start Date and End Date fields doesn't describe which date you're currently on until after you've selected it.
- Tabbing past the Additional Comments field will sometimes focus on the Select your Employer... button, and sometimes go directly into the Select your Employer... search field. When the latter happens it does not say which button you're focused on.
- Focusing on Add Honor/Memory Info does not give any information on what this field is.
- When filling out Honor/Memory information, there's no indicator that you're filling out the information for the address to send the acknowledgement.
